e2e/cypress/integration/api/iam/12_projects_api_node_ingestion.spec.ts

```diff
@@ -217,67 +217,92 @@ describeIfIAMV2p1('Nodemanager ingestion project tagging', () => {
     });
 
     // wait for the report to be ingested
-    waitForNodemanagerNode(complianceNodeId, 30);
+    cy.waitForNodemanagerNode(complianceNodeId, 30);
+    cy.waitForComplianceNode(complianceNodeId, start, end, 30);
     // wait for the client run report to be ingested
-    waitForNodemanagerNode(clientRunsNodeId, 30);
+    cy.waitForNodemanagerNode(clientRunsNodeId, 30);
+    cy.waitForClientRunsNode(clientRunsNodeId, 30);
```

Comment:
what are L223 and L224 doing differently? they look like they're waiting on the same client runs node.

Reply:
Line 223 is waiting for the node manager to ingest the node and line 224 is waiting for Client Runs ingestion. Before we were just waiting on the node manager it turns out that node manager is faster than Client Runs.